packages/sanity/src/core/form/components/formField/FormFieldHeader.tsx

```diff
@@ -2,19 +2,19 @@
 
 import React, {memo} from 'react'
 import {Box, Flex} from '@sanity/ui'
-import {FieldPresence} from '../../../presence'
-import {NodePresence, NodeValidation} from '../../types'
+import {NodeValidation} from '@sanity/types'
+import {FieldPresence, FormFieldPresence} from '../../../presence'
 import {FormFieldHeaderText} from './FormFieldHeaderText'
 
 export interface FormFieldHeaderProps {
   /**
-   * @alpha
+   * @beta
    */
   validation?: NodeValidation[]
```

Comment:
I agree. There's some namings to consolidate here. I think `NodeValidation` is too generic. How about:

- `presence: FormNodePresence`
- `validation: FormNodeValidation`

The reason to rename from `FormField` to `FormNode` is the "field" is not always accurate, since we have both _object fields_ and _array/list items_.

Reply:
cc @bjoerge

Reply:
For the record – I did not rename anything here, but I removed the `NodePresence` interface (it was just an alias to `FormFieldPresence`).

Reply:
I'm probably wrongly indexing on v2 -> v3 changes too: all import paths are changed anyway, so we can probably do whatever we want to be honest. But renaming interfaces makes it harder to find the replacement. Having said that, the *InputProps interfaces should guide users to the right thing.

This could also be an argument to hold for Type_Schema_Options if you would like to argue that ;)

Reply:
We're _not_ change the `*InputProps`! Those are solid, and well-named.

The `NodePresence` was introduced in v3, so shouldn't be a huge problem to redact.

Reply:
I support renaming from `FormField*` to `FormNode*`. Good call!

Reply:
Great, thanks for chiming in @bjoerge 🙏 Will do the change!